Assets/MRTK/Providers/WindowsMixedReality/XR2018/WindowsMixedRealityEyeGazeDataProvider.cs

```diff
@@ -136,17 +136,19 @@ public override void Update()
 #if (UNITY_WSA && DOTNETWINRT_PRESENT) || WINDOWS_UWP
             if (WindowsMixedRealityUtilities.SpatialCoordinateSystem == null || !eyesApiAvailable)
             {
+                Service?.EyeGazeProvider?.UpdateEyeTrackingStatus(this, null);
                 return;
             }
 
+            bool? isCalibrationValid = null;
```

Comment:
Why not just set it to false if it's a system that doesn't support eye tracking?

Reply:
My current understanding is that even on devices that do support eye tracking, if the device loses the eyes, then the EyesPose off the SpatialPointerPose goes null. So even on HL2 the EyesPose can go back to null.

I'm trying to catch on the HL2 when it loses eyes.